modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc

```diff
@@ -0,0 +1,79 @@
+= Converters and serialization 
+:description: Connectors Converters and serialization
+:page-cloud: true
+
+== Key and value converters
+
+Each Redpanda message is a key and value record. Converters handle serialization and deserialization of data between Redpanda topic and a connector. Record key and value converters are configured separately. It can be specified using `Redpanda message key format` and `Redpanda message value format` properties. Key and value converters can be different.
+
+[NOTE]
+====
+If an external system requires structured data (like BigQuery, Snowflake, SQL database), you need to provide data with schema. Then you need to use Avro Converter, Protobuf Converter or JSON Converter with schema.
```

Comment:
Snowflake is a little bit tricky. I think it does not require the schema actually.